web/src/routes/index.ts

```diff
@@ -28,6 +29,11 @@ const routes: Array<RouteConfig> = [
     path: '/maps',
     exact: true,
     component: Map
+  },
+  {
+    path: '/confirm-upload',
```

Comment:
Whether to use a dedicated path and what to call it is a standing discussion for Issue #219.